amplitude/management/commands/export_to_amplitude.py

```diff
@@ -42,7 +48,25 @@ def iter_events(cls, last_synced_at: datetime.datetime) -> Iterator[AmpEvent]:
                     "lastLogin": user.last_login,
                     "dateJoined": user.date_joined,
                     "adminUrl": absolute_reverse("admin:users_justfixuser_change", args=(user.pk,)),
+                    "agreedToJustfixTerms": oi.agreed_to_justfix_terms,
+                    "agreedToNorentTerms": oi.agreed_to_norent_terms,
+                    "agreedToEvictionfreeTerms": oi.agreed_to_evictionfree_terms,
+                    "canReceiveRttcComms": oi.can_receive_rttc_comms,
+                    "canReceiveSajeComms": oi.can_receive_saje_comms,
+                    "receivesPublicAssistance": oi.receives_public_assistance,
+                    "hasCalled311": oi.has_called_311,
+                    "hasAptNumber": bool(oi.apt_number),
+                    "zipcode": oi.zipcode,
+                    # These are also synced via the front-end, so we need to
+                    # be extra certain they're calculated the same way. See amplitude.ts
+                    # for more details.
+                    "city": oi.city,
+                    "state": oi.state,
+                    "signupIntent": oi.signup_intent,
+                    "leaseType": oi.lease_type,
+                    "isEmailVerified": user.is_email_verified,
                 },
+                insert_id_suffix=str(update_time),
```

Comment:
Amplitude ignores IDs it's seen in the past 7 days, so we explicitly want to make the update time part of the ID, so that it doesn't erroneously ignore real changes in our data.